config/samples/metallb.io_v1alpha1_addresspool.yaml

```diff
@@ -27,4 +27,25 @@ metadata:
 spec:
   protocol: layer2
   addresses:
-    - 2002:2:2::1-2002:2:2::100
\ No newline at end of file
+    - 2002:2:2::1-2002:2:2::100
+
+---
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample4
+  namespace: metallb-system
+spec:
+  protocol: bgp
+  addresses:
+    - 172.30.0.100-172.30.0.255
+  bgpAdvertisements:
+  -
```

Comment:
the empty line is supposed to be there?

Reply:
its valid that way and convient for the template to generate list of bgpadv